src/routing_table.rs

```diff
@@ -46,23 +46,23 @@ impl NodeInfo {
     }
 
     #[cfg(not(test))]
-    pub fn id(&self) -> &NameType {
+    pub fn name(&self) -> &NameType {
         self.public_id.name()
     }
 
     #[cfg(test)]
     pub fn new(public_id: PublicId, connections: Vec<Connection>) -> NodeInfo {
-        let id = public_id.name().clone();
+        let name = public_id.name().clone();
         NodeInfo {
             public_id: public_id,
             connections: connections,
-            id: id,
+            name: name,
         }
     }
 
     #[cfg(test)]
-    pub fn id(&self) -> &NameType {
-        &self.id
+    pub fn name(&self) -> &NameType {
+        &self.name
```

Comment:
it may be better to not store the name but make it `public_id.name().clone();` if this is in fact the name? or is this only the initial name? If only for tests as it seems we should consider this as we move on.